models/BusinessText.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+const Language = require('./Language').schema;
+
+const businessTextSchema = new mongoose.Schema({
+    code: String,
```

Comment:
Why do we need code?

Reply:
Since data for About Us, Terms & Conditions, etc seem to have the same simple structure, I made one collection for them all. Code ('about-us', 'terms-and-conditions') serves to tell the documents apart without diving into multilanguage components like title or relying on _id which is different each time I seed the DB. Code can also be used to dynamically generate web-address of the particular page

Reply:
Okay, got it